### PR TITLE
Help differentiate rollout deployment and restart steps

### PR DESCRIPTION
**Pull request by SushrutHarness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/kubernetes-rollout-restart.md [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+sidebar_position: 3
 ---
 
 :::info note
 1. This feature is currently behind the feature flag, `CDS_K8S_ROLLOUT_STEP`. Contact [Harness support](mailto:support@harness.io) to enable it.
 2. This feature requires Harness Delegate version **832xx**.
 :::
 
-This topic describe how you can restart your deployed Kubernetes resources by using **K8s Rollout** step.
+This topic describes the **K8s Rollout** step which allows you to manage your **Kubernetes Rolling Deployment**. In pipeline studio, this step will be labeled as **K8sRollout**. On inspection, the step is called **K8s Rollout**.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/kubernetes-rollout-restart.md:12]:
```suggestion
This topic describes the **K8s Rollout** step, which allows you to manage your **Kubernetes Rolling Deployment**. In Pipeline Studio, this step will be labeled as **K8sRollout**. On inspection, the step is called **K8s Rollout**. 
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/kubernetes-rollout-restart.md [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 :::
 
-This topic describe how you can restart your deployed Kubernetes resources by using **K8s Rollout** step.
+This topic describes the **K8s Rollout** step which allows you to manage your **Kubernetes Rolling Deployment**. In pipeline studio, this step will be labeled as **K8sRollout**. On inspection, the step is called **K8s Rollout**. 
 
-Harness follows standard Kubernetes behavior during rollout. See [kubectl rollout](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_rollout/).
+Harness follows standard Kubernetes behavior during rollout. To see what behavior this replicates, go to the subcommands of [kubectl rollout](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_rollout/#see-also).
 
-When to use **K8s Rollout** step?
+When should you use the **K8s Rollout** step?
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/kubernetes-rollout-restart.md:16]:
```suggestion
When to use the **K8s Rollout** step
```
Optional sug.

**Review by douglas-j-bothwell — DISMISSED** [review]
A few sugs otherwise LGTM

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/kubernetes-rollout-step.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+description: Define the Kubernetes Rollout Deploy step.
 sidebar_position: 2
 helpdocs_topic_id: 2bwlugh9gi
 helpdocs_category_id: 85tr1q4hin
 helpdocs_is_private: false
 helpdocs_is_published: true
 ---
 
-This topic provides settings and permissions for the Kubernetes Rollout Deployment step.
+This topic provides settings and permissions for the Kubernetes Rollout Deployment step. In pipeline studio, the step will be labeled as **Rollout Deployment**. On inspection, the step is called **K8s Rollout Deploy**.
```
Comment (SushrutHarness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/kubernetes-rollout-step.md:11]:
```suggestion
This topic provides settings and permissions for the Kubernetes Rollout Deployment step. In Pipeline Studio, the step will be labeled as **Rollout Deployment**. On inspection, the step is called **K8s Rollout Deploy**. 
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/kubernetes-rollout-restart.md [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
 :::info note
 1. This feature is currently behind the feature flag, `CDS_K8S_ROLLOUT_STEP`. Contact [Harness support](mailto:support@harness.io) to enable it.
 2. This feature requires Harness Delegate version **832xx**.
 :::
 
-This topic describe how you can restart your deployed Kubernetes resources by using **K8s Rollout** step.
+This topic describes the **K8s Rollout** step, which allows you to manage your **Kubernetes Rolling Deployment**. In Pipeline Studio, this step will be labeled as **K8sRollout**. On inspection, the step is called **K8s Rollout**. 
 
-Harness follows standard Kubernetes behavior during rollout. See [kubectl rollout](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_rollout/).
+Harness follows standard Kubernetes behavior during rollout. To see what behavior this replicates, go to the subcommands of [kubectl rollout](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_rollout/#see-also).
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/kubernetes-rollout-restart.md:14]:
```suggestion
Harness follows standard Kubernetes behavior during rollout. To see what behavior this replicates, go to the subcommands of [kubectl rollout](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_rollout/#see-also) in the Kubernetes documentation.
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/kubernetes-rollout-step.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+description: Define the Kubernetes Rollout Deploy step.
 sidebar_position: 2
 helpdocs_topic_id: 2bwlugh9gi
 helpdocs_category_id: 85tr1q4hin
 helpdocs_is_private: false
 helpdocs_is_published: true
 ---
 
-This topic provides settings and permissions for the Kubernetes Rollout Deployment step.
+This topic provides settings and permissions for the Kubernetes Rollout Deployment step. In Pipeline Studio, the step will be labeled as **Rollout Deployment**. On inspection, the step is called **K8s Rollout Deploy**.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/kubernetes-rollout-step.md:11]:
```suggestion
This topic provides settings and permissions for the Kubernetes Rollout Deployment step. In Pipeline Studio, the step is labeled **Rollout Deployment**. On inspection, the step is labeled **K8s Rollout Deploy**. 
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/kubernetes-rollout-restart.md [hunk 6]
```diff
@@ -1,18 +1,19 @@
...
+sidebar_position: 3
 ---
 
 :::info note
 1. This feature is currently behind the feature flag, `CDS_K8S_ROLLOUT_STEP`. Contact [Harness support](mailto:support@harness.io) to enable it.
 2. This feature requires Harness Delegate version **832xx**.
 :::
 
-This topic describe how you can restart your deployed Kubernetes resources by using **K8s Rollout** step.
+This topic describes the **K8s Rollout** step, which allows you to manage your **Kubernetes Rolling Deployment**. In Pipeline Studio, this step will be labeled as **K8sRollout**. On inspection, the step is called **K8s Rollout**.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/kubernetes-rollout-restart.md:12]:
```suggestion
This topic describes the **K8s Rollout** step, which allows you to manage your **Kubernetes Rolling Deployment**. In Pipeline Studio, this step is labeled **K8sRollout**. On inspection, the step is labeled **K8s Rollout**. 
```

**Review by brian-f-harness — APPROVED** [review]

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**